ldi-nifi/ldi-nifi-processors/geojson-to-wkt-processor/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/processors/config/GeoJsonToWktProcessorProperties.java

```diff
@@ -19,8 +21,20 @@ private GeoJsonToWktProcessorProperties() {
 			.addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
 			.build();
 
+	public static final PropertyDescriptor TRANSFORM_TO_RDF_WKT = new PropertyDescriptor.Builder()
+			.name("TRANSFORM_TO_RDF_WKT")
+			.displayName("Transform GeoJson to RDF+WKT format, defaults to false and the default format is WKT")
+			.required(false)
+			.defaultValue(FALSE.toString())
+			.addValidator(StandardValidators.BOOLEAN_VALIDATOR)
+			.build();
+
 	public static Lang getDataSourceFormat(final ProcessContext context) {
 		return RDFLanguages.nameToLang(context.getProperty(DATA_SOURCE_FORMAT).getValue());
 	}
 
+	public static boolean getTransformToRdfWkt(final ProcessContext context) {
+		return context.getProperty(TRANSFORM_TO_RDF_WKT).asBoolean();
```

Comment:
Can this be like this example (copy-pasted from another ProcessorProperties class), as this can handle both null and non-boolean cases
```java
TRUE.equals(context.getProperty(FORCE_CONTENT_TYPE).asBoolean());
```